README.md

```diff
@@ -149,6 +149,12 @@ A small team of analytics engineers at Fivetran develops these dbt packages. How
 
 We highly encourage and welcome contributions to this package. Check out [this dbt Discourse article](https://discourse.getdbt.com/t/contributing-to-a-dbt-package/657) on the best workflow for contributing to a package!
 
+## Opinionated Modelling Decisions
+
+The ServiceNow Connector provides thousands of tables. We developed our data models based on the tables with highest usage across Fivetran customers. 
```

Comment:
I don't think this is necessary as this is a decision we apply to all our data models.